tox.ini

```diff
@@ -28,6 +28,7 @@ commands =
 platform = linux
 commands =
     pipenv install --dev --skip-lock
+    pipenv run pip uninstall gevent gevent-websocket -y
```

Comment:
I thought we could use pipenv uninstall gevent ... (without run pip) ?

Reply:
it is equivalent in this case since we won't be using piplock